### ISSUE TITLE
`StatsConfig` does not have `beaker_dataset`

### ISSUE THREAD

**Issue by elynnwu** [ai2cm/ace#441]
`compute_dataset` currently has an error because `beaker_dataset` is not in `StatsConfig`:

```
config = DatasetConfig.from_file(config).dataset_computation
```

`beaker_dataset` is still in `upload_stats`. Is this intentional?

**Comment by elynnwu** [comment]
This is possibly only breaking e3smv2 workflow - need to double check

**Comment by AnnaKwa** [comment]
I just ran into this with the fv3gfs workflow too.

**Comment by AnnaKwa** [comment]
Also, after commenting out the beaker_dataset field, there was another issue where the compute_dataset script tried to import `get_stats` but the stats script was not available in that task. I had to pass the stats_script parameter to the `compute-fme-dataset-individual` task and write it locally before running the command for the main script:
```
...
            cat << EOF > get_stats.py
            {{inputs.parameters.stats_script}}
            EOF
            
            ...
```

**Comment by jpdunc23** [comment]
Also encountered this when testing `compute_dataset.py` for https://github.com/ai2cm/full-model/pull/1148. Commenting out `stats.beaker_dataset` allows me to run the `compute_dataset.py` script directly, but breaks `test_config.py`.

**Comment by oliverwm1** [comment]
This was resolved by ai2cm/full-model#1167
